### ISSUE TITLE
Permanent image feed disabling

### ISSUE THREAD

**Issue by MortalSage** [pythongosssss/ComfyUI-Custom-Scripts#184]
I currently disable the image feed on every launch, because I don't need it and it takes up lots of window space.
Can you add a default option "Minimized" for "Image Feed Location"?

Thank you.

**Comment by pythongosssss** [comment]
Once you close it, it will already remember the closed state
Do you have your browser clearing all data on close or similar?

Last time this was raised, the problem was that the image feed wasnt from this repo but from comfy_mtb so can you please confirm the image feed is from this repo - there should NOT be a load session button if it is mine

**Comment by MortalSage** [comment]
This is exactly my issue. My browser is set to not remember anything.

I tried to set an exception for comfy, but it didn't work. I also don't know what I would need to keep saved so the browser can remember.

This is why I'm currently looking for a solution that can save this options as default in the server, so that's why I asked if you could add an extra option.

I had neither mtb nor custom scripts installed before, but on comfyui repo, they said your repo could help, so I installed it.
I think the feed has now a new "resize feed" button and I can't see "load session" on the feed.

Also, due to this, I finally have a tag autocomplete, which is very good.

**Comment by steve-ttt** [comment]
It's quite disappointing that there isn't an option to turn this off. 

Just like @MortalSage, I also make sure to clear my browser cache when I exit (I prefer not to have meta and others tracking my online activities). For those who might find themselves in a similar situation, I've found a workaround by renaming the file to prevent the image feed from loading.

Here are the commands I used:

```bash
cd ComfyUI/custom_nodes/ComfyUI-Custom-Scripts/web/js
mv imageFeed.js imageFeed.js.old
```
